Classes/System/Configuration/TypoScriptConfiguration.php

```diff
@@ -568,6 +568,29 @@ public function getEnabledIndexQueueConfigurationNames($defaultIfEmpty = [])
     }
 
     /**
+     * Retrieves an array of additional Fields that will trigger an
```

Comment:
Retrieves an array of additional **f**ields that will trigger a~~n~~ recursive **update** of pages when some of the fields on that page are modified.